src/Microsoft.AspNet.Mvc.Core/OutputFormatterOptions.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNet.Mvc.Core
+{
+    public class OutputFormatterOptions
+    {
+        private Dictionary<string, MediaTypeHeaderValue> map = new Dictionary<string, MediaTypeHeaderValue>();
+
+        public void AddFormatMapping(string format, MediaTypeHeaderValue contentType)
+        {
+            if (!string.IsNullOrEmpty(format) && contentType != null)
+            {
+                if(format.StartsWith("."))
+                {
+                    format = format.TrimStart('.');
```

Comment:
what does `TrimStart` do if it doesn't start with a period? What does it do if it starts with multiple periods? `....foo`